content/release_notes_chefdk.md

```diff
@@ -2946,7 +2946,7 @@ This new release also includes three new matchers: `dnf_package`,
 `msu_package`, and `cab_package` and utilizes the new Fauxhai 4.0
 release. This release adds several new platforms and removes many older
 end-of-life platforms. See
-[PLATFORMS.md](https://github.com/customink/fauxhai/blob/master/PLATFORMS.md)
+[PLATFORMS.md](https://github.com/customink/fauxhai/blob/main/PLATFORMS.md)
```

Comment:
Another community repo, so confirm the branch. Or if it even exists.

Reply:
This exists with main.